lib/BackendClient.js

```diff
@@ -296,7 +295,7 @@ var BackendClient = (function(){
 
             deferred.resolve(res);
         }, filter);
-
+        client.setPartnerId(partnerId);
```

Comment:
I think better to add comment "impersonate" - for the first glance it is not clear the partnerId is the -5 
As in https://github.com/kaltura/liveDVR/blob/c21a05c07efcc7757c8e57592235d6f3ab4d498c/lib/BackendClient.js#L476